### PR TITLE
Works & About Section (mobile design)

### PR DESCRIPTION
**Pull request by mahammad-mostafa**
### Works Section:
- Combined grid layout with flexbox
- Grid layout used for cards structure
- Flexbox used for each card content
- Implemented buttons interactions

### About Section:
- Nested list structure
- Built with several flexbox layouts
- Special classes for shapes

### CODE REVIEW

File: index.html
Comment (Ofarouq310) [inline index.html:1]: - [ ] Kindly note that inline styles are not allowed, for the best `CSS` practices. I can see you have used them inside all of your `img` tags, kindly fix that.

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @mahammad-mostafa, ✋ 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

giphy


## Highlights
✅ Descriptive PR title & summary,
✅ Followed GitHub flow.
✅ Professional README file.


## Required Changes ♻️

- [ ] Kindly implement the button interactions provided in the `Figma` design.

   image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (mahammad-mostafa) [reply index.html:1]: Hi @Ofarouq310,

Thanks for your feedback. However, I have already implemented the button interactions with a class called `.button`. You can check the hover effect in `See Project` & `Get my resume` links.

Looking forward to your feedback,

Comment (Ofarouq310) [reply index.html:1]: Hello @mahammad-mostafa, 

I can't see them working on my side, I will address this issue, but kindly process with the other requirements to be able to ask for another review.

I wish you all the best.

Comment (mahammad-mostafa) [reply index.html:1]: Hey @Ofarouq310,

I have changed all inlined CSS in `img` tags. I am ready to push and request a review. However, I want your confirmation that the button interaction issue is resolved.

Thanks,

Comment (Ofarouq310) [reply index.html:1]: Hey @mahammad-mostafa,

Thanks for your patience, everything works perfectly fine on my side.

Happy coding.

**Review by jmonto55 — APPROVED** [review]
# STATUS: APPROVED 🟢

Hi @mahammad-mostafa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

AgreeSealGIF


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**